### DISCUSSION TITLE
客户端配置的 URL 格式是否有个标准？

### DISCUSSION THREAD

**Discussion by cupen** [v2fly/v2ray-core#1791] [Ideas]
我手机上的不同客户端生成的 URL 似乎互相不兼容。
所以我想开发个工具根据客户端配置生成 URL ，有个官方标准比较好。

**Comment by Cwek** [comment]
这个好像讨论了很久，就是core不需要提供这些功能，由外壳自己去指定配置打包方案。

**Reply by cupen** [reply]
有其它类似讨论的入口么？ 我想站个队。
这事确实不需要在 core 项目里维护，但由大腿牵头来钦定个标准是很有价值的，可以免于客户端重复做无用功。
